pkgs/tools/system/lshw/default.nix

```diff
@@ -1,27 +1,33 @@
-{ stdenv, fetchurl
-, withGUI? false, gtk? null, pkgconfig? null, sqlite? null  # compile GUI
- }:
-stdenv.mkDerivation rec {
+{ stdenv, lib, fetchurl, fetchpatch
+, withGUI ? false, gtk2, pkgconfig, sqlite # compile GUI
```

Comment:
Optional dependencies should have `? null`. Otherwise, they must be passed even if not needed.